app/src/main/res/layout/item_view.xml

```diff
@@ -3,21 +3,22 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:minHeight="@dimen/item_height"
-    android:padding="@dimen/small_padding">
+    android:minHeight="56dp"
```

Comment:
Do not hardcode values in layout. Better move the value to previously used `@dimen/item_height`